### PR TITLE
Create international-journal-of-polymer-analysis-and-characterization…

### PR DESCRIPTION
**Pull request by POBrien333**
….csl

via https://forums.zotero.org/discussion/77228/add-a-journal-to-the-styles#latest

### CODE REVIEW

**Comment by adam3smith** [comment]
Thanks!

**Comment by POBrien333** [comment]
Thanks for the fix.

File: international-journal-of-polymer-analysis-and-characterization.csl [hunk 1]
```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>International Journal of Polymer Analysis and Characterization</title>
+    <title-short>IJPAC</title-short>
+    <id>http://www.zotero.org/styles/international-journal-of-polymer-analysis-and-characterization</id>
+    <link href="http://www.zotero.org/styles/international-journal-of-polymer-analysis-and-characterization" rel="self"/>
+    <link href="http://www.zotero.org/styles/environmental-toxicology-and-chemistry" rel="template"/>
+    <link href="http://onlinelibrary.wiley.com/journal/10.1002/%28ISSN%291552-8618/homepage/ForAuthors.html" rel="documentation"/>
```
Comment (rmzelle) [inline international-journal-of-polymer-analysis-and-characterization.csl]: @POBrien333, updated this documentation link, which pointed to the template style journal (and gives a 404), in https://github.com/citation-style-language/styles/pull/4213.

Comment (POBrien333) [reply international-journal-of-polymer-analysis-and-characterization.csl]: Thank you.

**Pull request merged**